official/resnet/ctl/ctl_imagenet_benchmark.py

```diff
@@ -85,7 +128,7 @@ def _get_model_dir(self, folder_name):
     return os.path.join(self.output_dir, folder_name)
 
 
-class Resnet50CtlBenchmarkBase(ctl_benchmark.CtlBenchmark):
+class Resnet50CtlBenchmarkBase(CtlBenchmark):
```

Comment:
these tests should set the warmup steps, otherwise we may see some slow perf in the beginning?

Reply:
Done.